internal/models/running_output.go

```diff
@@ -195,15 +206,15 @@ func (ro *RunningOutput) WriteBatch() error {
 func (ro *RunningOutput) Close() {
 	err := ro.Output.Close()
 	if err != nil {
-		log.Printf("E! [outputs.%s] Error closing output: %v", ro.Name, err)
+		log.Printf("E! [%s] Error closing output: %v", ro.LogName(), err)
 	}
 }
 
 func (ro *RunningOutput) write(metrics []telegraf.Metric) error {
 	dropped := atomic.LoadInt64(&ro.droppedMetrics)
 	if dropped > 0 {
-		log.Printf("W! [outputs.%s] Metric buffer overflow; %d metrics have been dropped",
-			ro.Name, dropped)
+		log.Printf("W! [%s] Metric buffer overflow; %d metrics have been dropped",
```

Comment:
Use the plugin's logger in the RunningFoo's

Reply:
:grey_question: